### ISSUE TITLE
It doesn't run the tests after the WebDriver starts

### ISSUE THREAD

**Issue by jourdanrodrigues** [rochejul/gulp-angular-protractor#27]
This is my gulp task, which is working fine but running the tests:

```
// Setting up the test task
gulp.task("protractor", [fileHandlerTask, "connect"], function(callback) {
    gulp
        .src(["tests/menu-spec.js"])
        .pipe(gulpProtractorAngular({
            configFile: "tests/conf.js",
            debug: false,
            autoStartStopServer: true
        }))
        .on("error", function(error) {
            console.log(error);
        })
        .on("end", callback);
});
```

**Comment by rochejul** [comment]
Hi @jourdanrodrigues 
What is the content of your tests/conf.js file ? Maybe there is missing some configuration ?

Can you provide too your context:
- OS
- Node version
- NPM version
- gulp-angular-protractor version

And give me your shell output

Thanks

**Comment by jourdanrodrigues** [comment]
My `tests/config.js`:

```
var PORT = process.env.PORT;
PORT = typeof(PORT) != "undefined" && PORT != "" ? PORT : 8888;

exports.config = {
    baseUrl: "http://localhost:" + PORT + "/",
    rootElement: ".controk", // http://stackoverflow.com/a/22518925
    directConnect: true,
    specs: ["*-spec.js"]
};
```

Where the main issue happened:

```
OS: Ubuntu 16.10;
Node version: 6.9.1;
NPM version: 3.10.8;
gulp-angular-protractor version: 0.2.0.
Shell output: Stops at `[HH:MM:SS] [HH:MM:SS] I/start - seleniumProcess.pid: <PID>`.
```

I just ran the tests on the following environment and it executed fine, but I noticed that in case of test error, it doesn't throw the test status code, it throws 0 from Selenium WebDriver (which will be useless since I want to integrate with Travis CI and it needs the "not OK" status to see that the tests failed):

```
OS X El Capitan 10.11.6;
Node version: 6.5.0;
NPM version: 3.10.3;
gulp-angular-protractor version: 0.2.0.
```

Thanks for the quick response. :)

**---- UPDATE 1 ----**

About the MacOS issue, I took a little time to read the docs and found the "error" event, which through RegExp I managed to end the process with the status code sent. Sorry for posting it without trying.

The Ubuntu issue I'll still see later.

**---- UPDATE 2 ----**

Updated info about Ubuntu problem.

**---- UPDATE 3 ----**

I managed to run the tests with Travis CI on the above Ubuntu settings (but on Ubuntu 14.04) and they're executing fine. Seems like the problem is at Ubuntu 16.10.
